notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>There is a lot of new code in the next block, so it would be good explain in more words here what will happen.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>